nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -10,8 +10,7 @@ in {
 
   # Required to provide good EBS experience,
   # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html#timeout-nvme-ebs-volumes
-  # TODO change value to 4294967295 when kernel is updated to 4.15 or later
-  config.boot.kernelParams = [ "nvme_core.io_timeout=255" ];
+  config.boot.kernelParams = [ "nvme_core.io_timeout=4294967295" ];
```

Comment:
How about guarding this?

```
nix-repl> pkgs.lib.versionAtLeast config.boot.kernelPackages.kernel.version "4.15"
true
```

Reply:
Updated accordingly, appears to be working on my 20.03 build:

```
$ nix show-derivation /nix/store/dnm13wvmrxp7w1jmf43jr2sls7z9idg5-nixos-system-unnamed-20.09pre130979.gfedcba.drv | grep 'kernelParams"'
      "kernelParams": "panic=1 boot.panic_on_fail console=ttyS0 nvme_core.io_timeout=4294967295 loglevel=4",
```